### PR TITLE
add function delete and edit comment

### PR DESCRIPTION
**Pull request by manh2911**

### CODE REVIEW

**Comment by manh2911** [comment]
ready

**Comment by manh2911** [comment]
ready

**Comment by manh2911** [comment]
ready

**Comment by manh2911** [comment]
ready

**Comment by manh2911** [comment]
ready

**Comment by manh2911** [comment]
ready

**Comment by manh2911** [comment]
ready

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -18,13 +18,14 @@ public function updateProfile($id,Request $request)
             $fileName = $file->getClientOriginalName('image');
             $path = 'image';
             $file->move($path, $fileName);
-            $user = User::where('id', $id)->update(['image' => $fileName]);
+            $realFileName = 'image/'.$fileName;
```
Comment (BunnyPi04) [inline app/Http/Controllers/UserController.php]: convention

File: resources/js/review_image.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    $(".removeimg").show();
+    $(".Choicefile").unbind("click"); //Xóa sự kiện  click trên nút .Choicefile
+}
+    $(".Choicefile").bind("click", function  () 
+    { //Chọn file khi .Choicefile Click
+        $("#uploadfile").click();         
+    });
+    $(".removeimg").click(function () {//Xóa hình  ảnh đang xem
+        $("#thumbimage").attr("src", "").hide();
+        $("#myfileupload").html("<input type='file' id='uploadfile'  onchange='readURL(this);' />");
```
Comment (BunnyPi04) [inline resources/js/review_image.js]: `id='uploadfile'  onchange='readURL(this);'`
dư space

File: resources/js/review_image.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+//////// xem truoc anh
+function  readURL(input,thumbimage) {
```
Comment (BunnyPi04) [inline resources/js/review_image.js:2]: `function  readURL`
dư 1 space

File: resources/js/review_image.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    var  reader = new FileReader();
```
Comment (BunnyPi04) [inline resources/js/review_image.js:3]: đây nữa

**Review by BunnyPi04 — COMMENTED** [review]
sửa pull sau nhé

File: resources/js/review_image.js (hunk 4 above)
Comment (manh2911) [reply resources/js/review_image.js:3]: vâng chị.

**Pull request merged**